run_dir/static/js/base.js

```diff
@@ -248,10 +248,15 @@ function nice_numbers (count) {
 function auto_samples_cell (id, val){
   // Column returns an array
   if (val instanceof Array){
-    cell = '<td class="' + id + '">';
-    $.each(val, function(key, val){
-      cell += auto_format(val, true) + ' ';
-    });
+    if (val.length === 0){
+      cell = '<td class="' + id + '">' + '-';
+    }
+    else {
+      cell = '<td class="' + id + '">';
+      $.each(val, function(key, val){
+        cell += auto_format(val, true) + ' ';
+      });
+    }
```

Comment:
Oh, spoke too soon, it's not working, it's not giving me a dash for prep status 🤔  
OK, I'll look into why....

Reply:
The `$.each(val, function(key, val){` doesn't take the empty arrays into consideration

Reply:
Then this could work.
```suggestion
    cell = '<td class="' + id + '">' 
    if (val.length === 0)
      cell = cell + '-';
    $.each(val, function(key, val){
        cell += auto_format(val, true) + ' ';
    });
    
```

Reply:
That works! 👏